Source/Scene/Model.js

```diff
@@ -991,6 +991,42 @@ define([
             get : function() {
                 return this._upAxis;
             }
+        },
+
+        /**
+         * Gets the model's vertex memory in bytes. This includes all vertex and index buffers.
+         *
+         * @private
+         */
+        vertexMemoryInBytes : {
```

Comment:
When a model is cached, 3D Tiles should only report the memory for one model even if, for example, two b3dm tiles reference it.